.env.example

```diff
@@ -11,6 +11,5 @@ VITE_PROXY="wss://p.mutinywallet.com"
 VITE_ESPLORA="https://mutinynet.com/api"
 VITE_LSP="https://signet-lsp.mutinywallet.com"
 VITE_RGS="https://rgs.mutinynet.com/snapshot/"
-VITE_SELFHOSTED="true"
```

Comment:
we probably wanna keep this for free mutiny+, no?

Reply:
yeah I thought about that but we don't have it wired up rn... guess I can leave in the env